### DISCUSSION TITLE
Custom fields for WooCommerce Product Variations

### DISCUSSION THREAD

**Discussion by JoryHogeveen** [pods-framework/pods#6601] [Ideas]
## Feature

It would be awesome if we could support custom fields for product variations.

## Examples elsewhere

Code:
https://www.businessbloomer.com/woocommerce-add-custom-field-product-variation/
https://wordpress.stackexchange.com/questions/276941/woocommerce-add-extra-field-to-variation-product

**Comment by sc0ttkclark** [comment]
Work was done in 2.8 to enable more capabilities to do this sort of thing. Will be revisiting in the near future, probably 2022
